subsys/modbus/Kconfig

```diff
@@ -44,7 +44,6 @@ config MODBUS_SERIAL
 	default y
 	depends on SERIAL && SERIAL_HAS_DRIVER
 	depends on DT_HAS_ZEPHYR_MODBUS_SERIAL_ENABLED
-	select UART_USE_RUNTIME_CONFIGURE
```

Comment:
This is a breaking change and is not acceptable.
You may want to look at the Kconfig documentation to find an [alternative](https://docs.zephyrproject.org/latest/build/kconfig/tips.html#the-imply-statement).

Reply:
I'd say using `imply` instead of `select` along with the proposed code changes in the PR should cover it? What am I missing?

Reply:
This conversation applies to an older version of the patch.